design-system/materials/internals/definition.yaml

```diff
@@ -53,14 +53,18 @@ choiceGroupsByTheme:
         color-neutral-60: 'hsl(0, 0%, 60%)'
         color-neutral-90: 'hsl(0, 0%, 90%)'
         color-neutral-95: 'hsl(0, 0%, 95%)'
+        color-neutral-85: 'hsl(0, 0%, 85%)'
         color-info: '#078cdf'
         color-info-85: 'hsl(203.05555555555554, 93.9130434783%, 85%)'
         color-info-95: 'hsl(203.05555555555554, 93.9130434783%, 95%)'
         color-warning: '#f16d0e'
         color-warning-25: 'hsl(25.110132158590307, 89.0196078431%, 25%)'
         color-warning-95: 'hsl(25.110132158590307, 89.0196078431%, 95%)'
+        color-warning-85: 'hsl(25.110132158590307, 89.0196078431%, 85%)'
         color-error: '#e60050'
         color-error-95: 'hsl(339.1304347826087, 100%, 95%)'
+        color-error-85: 'hsl(339.1304347826087, 100%, 85%)'
+        color-error-40: 'hsl(339.1304347826087, 100%, 40%)'
```

Comment:
Nit: can you sort them in the correct order?

Reply:
Thank you for the feedback. 
Updated here: https://github.com/commercetools/ui-kit/pull/2370/commits/814bec57f29025943e58025de6e695be3f4e1ec6